integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -5043,6 +5047,125 @@ msgstr "Chat"
 msgid "Send message"
 msgstr "Nachricht abschicken"
 
+#: cms/templates/contacts/contact_list.html
+msgid "Archived Contacts"
+msgstr "Archivierte Kontakte"
+
+#: cms/templates/contacts/contact_list.html
+msgid "Back to contacts"
+msgstr "Zurück zu Kontakte"
+
+#: cms/templates/contacts/contact_list.html
+msgid "Archived contacts"
+msgstr "Archivierte Kontakte"
+
+#: cms/templates/contacts/contact_list.html
+#: cms/templates/pages/_page_xliff_import_diff.html
+#: cms/templates/push_notifications/push_notification_list.html
+msgid "Title"
+msgstr "Titel"
+
+#: cms/templates/contacts/contact_list.html
+#: cms/templates/languages/language_form.html
+#: cms/templates/offertemplates/offertemplate_list.html
+#: cms/templates/organizations/organization_list.html
+#: cms/templates/poicategories/poicategory_list.html
+#: cms/templates/regions/region_list.html cms/templates/roles/role_list.html
+msgid "Name"
+msgstr "Name"
+
+#: cms/templates/contacts/contact_list.html
+msgid "Name of related POI"
```

Comment:
```suggestion
msgid "Name of related location"
```

Do we have a naming guide how to call pois in english? I find it a bit confusing that the table says poi, while the sidebar says location.
However, I noticed that we have quite a lot of occurrences of "poi" in this file, as well as a lot of occurrences of "location"